xclim/sdba/adjustment.py

```diff
@@ -1855,17 +1862,25 @@ def _adjust(
         adj_kws.setdefault("interp", self.interp)
         adj_kws.setdefault("extrapolation", self.extrapolation)
 
-        g_idxs, gw_idxs = grouped_time_indexes(ref.time, self.group)
-        ds = self.ds.copy()
-        ds["g_idxs"] = g_idxs
-        ds["gw_idxs"] = gw_idxs
+        af_q = self.ds.af_q
+        rot_matrices = self.ds.rot_matrices
+        group = self.group
 
-        # adjust (adjust for npft transform, train/adjust for univariate bias correction)
-        out = mbcn_adjust(
-            ref=ref,
-            hist=hist,
-            sim=sim,
-            ds=ds,
+        times = get_times(hist.time, group=group)
+        af_q[group.prop] = times.times.values
+        af_q = af_q.rename({group.prop: "time"})
+        af_q = af_q.broadcast_like(hist)
+        # a bit dirty, should avoid this situation before
+        af_q = af_q.chunk({"time": -1})
```

Comment:
this is a hack: broadcast PROP dimension of af_q to the the full time dimension, so it can be used in `map_groups`. Perhaps there is already a built-in way to achieve this?